build-deploy.sh

```
@@ -0,0 +1,96 @@
+# Extract arguments
+ENV="$1"
+HOST="$2"
+SUBDOMAIN="$3"
+ENABLE_BASIC_AUTH=""
+
+# Parse remaining arguments
+shift 3
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --enable_basic_auth)
+            ENABLE_BASIC_AUTH="--enable_basic_auth"
```

Shellcheck (0.10.0)

[warning] ENABLE_BASIC_AUTH appears unused. Verify use (or export if used externally).

(SC2034)